# HG changeset patch
# Date 1363331883 0
# User sbr <sebastien@brissaud.name>
# Parent 3eda1ff84d8bbeed7502bbefdddf75048cc8d8bd
When opening the BuildObjectWindow, ensure the selected object is visible in the object selector.
diff --git a/src/object_gui.cpp b/src/object_gui.cpp
--- a/src/object_gui.cpp
+++ b/src/object_gui.cpp
@@ -58,6 +58,7 @@
 		NWidgetMatrix *matrix = this->GetWidget<NWidgetMatrix>(WID_BO_SELECT_MATRIX);
 		matrix->SetScrollbar(this->GetScrollbar(WID_BO_SELECT_SCROLL));
 		matrix->SetCount(ObjectClass::Get(_selected_object_class)->GetUISpecCount());
+		matrix->SetClicked(_selected_object_index);
 	}

 	virtual ~BuildObjectWindow()
